client/signup/steps/design-picker/index.jsx

```diff
@@ -7,8 +7,16 @@ import { connect } from 'react-redux';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import StepWrapper from 'calypso/signup/step-wrapper';
 import { submitSignupStep } from 'calypso/state/signup/progress/actions';
+import { getRecommendedThemes as fetchRecommendedThemes } from 'calypso/state/themes/actions';
+import { getRecommendedThemes } from 'calypso/state/themes/selectors';
 import './style.scss';
 
+// Ideally this data should come from the themes API, maybe by a tag that's applied to
+// themes? e.g. `link-in-bio` or `no-fold`
+const STATIC_PREVIEWS = [ 'bantry', 'sigler', 'miller', 'pollard', 'paxton', 'jones', 'baker' ];
```

Comment:
Does this equate to whether or not there's a pattern toolkit site for the theme?

Reply:
These are the themes that shouldn't have the hover effect e.g. the intention for Bantry is that everything appears above the fold. The design picker calls this kind of preview "static"
Adding a tag for this sort of theme is discussed here: p1632436419100900-slack-C029SB8JT8S